src/effects/effectchainslot.cpp

```diff
@@ -258,23 +254,30 @@ void EffectChainSlot::enableForInputChannel(const ChannelHandleAndGroup& handle_
     request->EnableInputChannelForChain.pEffectStatesMapArray = pEffectStatesMapArray;
 
     m_pEffectsManager->writeRequest(request);
-    slotChainChannelStatusChanged(handle_group.name(), true);
+
+    // TODO(Kshitij) : remove
+    clock_t elapsed = clock() - start;
+    qDebug() << "Elapsed time = " << elapsed << " = " << elapsed / CLOCKS_PER_SEC << "seconds.";
+    qDebug() << "ENABLED " << handle_group.name();
 }
 
 bool EffectChainSlot::enabledForChannel(const ChannelHandleAndGroup& handle_group) const {
```

Comment:
This function is unused and this information shouldn't be exposed to other classes. Remove it.